### ISSUE TITLE
Schedule.fixed not implemented

### ISSUE THREAD

**Issue by tylermk04** [zio/zio#1295]
I was hoping to use the `zio.Schedule#fixed` functionality after coming across it in [this presentation](https://www.slideshare.net/jdegoes/zio-schedule-conquering-flakiness-recurrence-with-pure-functional-programming-119932802). It appears that Schedule currently does not implement the `ZSchedule#fixed` trait (2.12-1.0.0-RC10-1).

```
[error] path/to/source:8:49: value fixed is not a member of object zio.Schedule
```

I figured it may have been intentionally removed (the presentation is from October), but then I noticed it's still found on the `ZSchedule` trait, so I figured I would ask. Thanks!

**Comment by ghostdogpr** [comment]
@tylermk04 what changed since the presentation is the introduction of ZIO Environment R. `Schedule` is just a particular type of `ZSchedule` with no environment requirement (R = Any).

`fixed` requires a `Clock` (it returns a `ZSchedule[Clock, Any, Int]`), that is why it is only available on `ZSchedule` and not on `Schedule`. `Schedule` only has the functions that don't require any environment.

In short, just use `ZSchedule.fixed` :)
